tests/framework/CodeInterfaceTests/MOOSEBaseApps/input_parser_csv.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/CodeInterfaceTests/MOOSEBaseApps.InputParser</name>
+    <author>talbpaul</author>
+    <created>2020-04-06</created>
```

Comment:
is this a file relocated or a new file? Update authorship and date if it's new.

Reply:
Updated.